src/utils/webpack/manifest-builder.js

```diff
@@ -1,6 +1,36 @@
 const fs = require("fs");
 const path = require("path");
 
+function parseShortcutRepoUrl(shortcutUrl) {
+    const match = shortcutUrl.match(/(?:([a-z]+):)?(.*)/i);
+
+    if (!match) {
+        return;
+    }
+
+    const [type, id] = match.slice(1);
+
+    switch (type) {
+        case "gist":
+            return `https://gist.github.com/${id}`;
+        case "bitbucket":
+            return `https://bitbucket.org/${id}`;
+        case "gitlab":
+            return `https://gitlab.com/${id}`;
+        case "github":
+        default:
```

Comment:
Default might be confusing here. Should we link to GitHub, if someone adds a random type? I'd prefer not displaying it at all.

Reply:
The default case is supposed to handle `npm/npm` format mentioned in the docs. However, we can strictly check it so that it can be displayed only if it conforms to the repo name format.

Reply:
Makes sense. 👍